src/Class.js

```diff
@@ -1,10 +1,10 @@
 var WebJack = {};
 
-(function(exports){
```

Comment:
I think it's miscopied, originally it's from this: http://caolan.org/posts/writing_for_node_and_the_browser/ -- can you get that working? Or is there a better best-practice?